frontend/package-lock.json

```diff
@@ -1,17820 +1,8 @@
 {
   "name": "t4cclient",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You're using an old version of `npm`. Is there a neeed to update the `package-lock.json`?

Reply:
I remove package-lock.json from the PR.